### PR TITLE
Add Python package documentation

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## Context

- Add documentation for the Python package separately
  - Add instructions for installation and basic usage - this can be fleshed out in detail.
- Update the documentation link to point to `commands/docs.md`

### CODE REVIEW

File: commands/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# DBT Copilot Tools
+
+This package contains a set of tools in the form of a Command Line Interface (CLI) for transferring applications/services from [GOV.UK PaaS](https://www.cloud.service.gov.uk) to Department for Business and Trade (DBT) PaaS which augments [AWS Copilot](https://aws.github.io/copilot-cli/).
```
Comment (WillGibson) [inline commands/README.md:3]: I think some of the commands will also be used in an ongoing way, not just for migration, as an application's requirements evolve, like if they needed an additional addon, or possibly to apply best practice changes to the CloudFormation templates.

**Review by WillGibson — COMMENTED** [review]
Looking good. One small comment, which may apply to the main `README.md` too.

**Review by gabelton — APPROVED** [review]

File: commands/README.md (hunk 1 above)
Comment (yusufsheiqh) [reply commands/README.md]: Updated in https://github.com/uktrade/copilot-tools/pull/77/commits/6dc71e41c39acfb30e07910ac5af40024f23e5bc. Thoughts?

Comment (WillGibson) [reply commands/README.md]: Looks good.

**Review by WillGibson — APPROVED** [review]

**Pull request merged**